### PR TITLE
Disable sorting for action column

### LINKED ISSUE #4061: Cannot sort by action in repo overview
In my opinion the action can just be removed for that column.

image

image

### PR DESCRIPTION
**Pull request by FreHu**
Resolves #4061

### CODE REVIEW

**Comment by larshp** [comment]
The

```abap
ELSE.
  IF something.
  ELSE.
  ENDIF.
ENDIF.
```
can be changed to 

```abap
ELSEIF something.
ELSE.
ENDIF.
```

File: src/ui/zcl_abapgit_gui_chunk_lib.clas.abap [hunk 1]
```diff
@@ -741,7 +746,7 @@ CLASS zcl_abapgit_gui_chunk_lib IMPLEMENTATION.
       lo_repo_online ?= io_repo.
       IF iv_show_branch = abap_true.
         IF iv_branch IS INITIAL.
-          ri_html->add( render_branch_span( iv_branch      = lo_repo_online->get_selected_branch( )
```
Comment (FreHu) [inline src/ui/zcl_abapgit_gui_chunk_lib.clas.abap:744]:
```suggestion
          ri_html->add( render_branch_span( iv_branch      = lo_repo_online->get_selected_branch( )
```

File: src/ui/zcl_abapgit_gui_chunk_lib.clas.abap [hunk 2]
```diff
@@ -780,7 +785,7 @@ CLASS zcl_abapgit_gui_chunk_lib IMPLEMENTATION.
           lv_display_url       TYPE zif_abapgit_persistence=>ty_repo-url,
           lv_icon_commit       TYPE string.
 
-    lv_commit_hash = io_repo_online->get_current_remote( ).
```
Comment (FreHu) [inline src/ui/zcl_abapgit_gui_chunk_lib.clas.abap:783]:
```suggestion
    lv_commit_hash = io_repo_online->get_current_remote( ).
```

**Review by larshp — APPROVED** [review]
looks good to me, thanks 👍

**Pull request merged**